shared/chat/conversation/messages/attachment/file/index.js

```diff
@@ -69,6 +77,7 @@ const styles = Styles.styleSheetCreate({
     position: 'absolute',
     right: 0,
   },
+  fail: {color: Styles.globalColors.red},
```

Comment:
Why not error?

Reply:
you mean why not name it "error" instead of "fail"? I just copied it from somewhere else. Can do